cmd/witnessctl/witnessctl.go

```diff
@@ -79,6 +80,7 @@ func init() {
 
 	flag.StringVar(&conf.hidPath, "d", "", "HID path of witness device to act upon (use -s to list devices)")
 	flag.BoolVar(&conf.status, "s", false, "get witness status")
+	flag.BoolVar(&conf.logs, "l", false, "get witness logs")
```

Comment:
Definitely add "console" or similar here, because if someone asked me what the "witness logs" were, I'd definitely say it meant the logs being witnessed.